lib/slack/real_time/concurrency/celluloid.rb

```diff
@@ -39,7 +39,7 @@ def run_loop
             logger.debug("#{self.class}##{__method__}") { e }
             driver.emit(:close, WebSocket::Driver::CloseEvent.new(1001, 'server closed connection')) unless @closing
           ensure
-            terminate
+            terminate if ::Celluloid::Actor.current.alive?
```

Comment:
Is `::Celluloid::Actor.current` the same thing as `self` here? In which case `terminate unless alive?` would work better.